### PR TITLE
Close #LGVISIUM-92: Add Github workflow to match the deployment expectations from Geowerkstatt

### PR DESCRIPTION
**Pull request by dcleres**
This PR makes it possible to generate the Docker Container of the Borehole data extraction API in a similar way to https://github.com/swisstopo/swissgeol-boreholes-suite. 

Upon merging to the main, a GitHub Actions creates automatically creates a new container with the latest version of the code and an `edge` tag. The container is then stored in the GitHub Container Registry.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            -f body="$(./get-changelog.sh)"
+
+      - name: Checkout new branch and patch changelog
+        run: |
+          git checkout -b $GIT_BRANCH_NAME
+          sed -i "/^\#\# \[Unreleased\]$/a \\\n\#\# $TAG_NAME - $(date '+%Y-%m-%d')" CHANGELOG.md
+
+      - name: Commit, push and create pull request
+        run: |
+          git config --global user.email "office@geowerkstatt.ch"
```
Comment (dcleres) [inline .github/workflows/release.yml:94]: @stijnvermeeren-swisstopo is there a similar address at SwissTopo that I could use?

File: .github/workflows/pre-release.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+      - name: Collect Docker image metadata (api)
+        id: meta-api
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ env.BASE_IMAGE_NAME }}-api
+          labels: |
+            org.opencontainers.image.created=${{ env.COMMITED_AT }}
+            org.opencontainers.image.version=v${{ env.VERSION }}
+            org.opencontainers.image.maintainer=Stijn Vermeeren <stijn.vermeeren@swisstopo.ch>
```
Comment (dcleres) [inline .github/workflows/pre-release.yml:37]: Is it okay to provide your address here? I could also use a generic contact address at SwissTopo

File: README.md [hunk 3]
```diff
@@ -383,6 +383,45 @@ If you have the AWS credentials configured locally in the `~/.aws` file, you can
...
+1. Pull the Docker Image
+
+```bash
+docker pull ghcr.io/swisstopo/swissgeol-boreholes-dataextraction-api:edge
+```
+
+2. a. Run the docker image from the Terminal
+
+```bash
+docker run -d --name swissgeol-boreholes-dataextraction-api -e AWS_ACCESS_KEY_ID=XXX -e AWS_SECRET_ACCESS_KEY=YYY -e AWS_ENDPOINT=ZZZ -p 8080:8080 ghcr.io/swisstopo/swissgeol-boreholes-dataextraction-api:TAG
```
Comment (flenny) [inline README.md:399]: JFYI: The `AWS_ENDPOINT` environment variable is not used yet when generating the S3 URL [here](https://github.com/swisstopo/swissgeol-boreholes-dataextraction/blob/2771506c69bcfcc159209fa1cd8b60b0a0706036/src/app/api/v1/endpoints/create_pngs.py#L52).

Comment (dcleres) [reply README.md:399]: Thank you for raising this issue. There is an issue about that that is about to get merged, so the `AWS_ENDPOINT` environment variable will be used in this position. As this ticket will be merged before this one, I chose to already make the code compatible.

**Review by flenny — APPROVED** [review]

**Comment by dcleres** [comment]
The CI was failing because the code was coming from a fork. Therefore, I opened a new branch with the same code and the PR can be found here: https://github.com/swisstopo/swissgeol-boreholes-dataextraction/pull/93

**Pull request closed without merging**